baseplate/clients/sqlalchemy.py

```diff
@@ -156,30 +159,50 @@ class SQLAlchemyEngineContextFactory(ContextFactory):
 
     """
 
-    PROM_PREFIX = "bp_sqlalchemy_pool"
-    PROM_LABELS = ["pool"]
+    PROM_PREFIX = "databasebp"
+    PROM_POOL_PREFIX = f"{PROM_PREFIX}_pool"
+    PROM_POOL_LABELS = ["pool"]
 
     max_connections_gauge = Gauge(
-        f"{PROM_PREFIX}_max_size",
+        f"{PROM_POOL_PREFIX}_max_size",
         "Maximum number of connections allowed in this pool",
-        PROM_LABELS,
+        PROM_POOL_LABELS,
     )
 
     checked_in_connections_gauge = Gauge(
-        f"{PROM_PREFIX}_idle_connections",
+        f"{PROM_POOL_PREFIX}_idle_connections",
         "Number of available, checked in, connections in this pool",
-        PROM_LABELS,
+        PROM_POOL_LABELS,
     )
 
     checked_out_connections_gauge = Gauge(
-        f"{PROM_PREFIX}_active_connections",
+        f"{PROM_POOL_PREFIX}_active_connections",
         "Number of connections in use, or checked out, in this pool",
-        PROM_LABELS,
+        PROM_POOL_LABELS,
     )
 
     overflow_connections_gauge = Gauge(
-        f"{PROM_PREFIX}_overflow_connections",
+        f"{PROM_POOL_PREFIX}_overflow_connections",
         "Number of connections over the desired size of this pool",
+        PROM_POOL_LABELS,
+    )
+
+    PROM_LABELS = [
```

Comment:
If we include prefix above we probably want to include it here too.

Reply:
I think differentiating information about the connection pool and the queries is fine, as they are different things. Not sure what you'd use the pool label for? Most of the time it will just be "sqlalchemy".